rebalance/rebalance.py

```diff
@@ -408,10 +412,20 @@ def maybe_rebalance_once(plugin: Plugin, failed_pairs: list):
     return {"success": False, "fee_spent": Millisatoshi(0)}
 
 
+def feeadjuster_toggle(plugin: Plugin, new_value: bool):
+    try:
+        msg = plugin.rpc.feeadjuster_toggle(new_value)
+        return msg["forward_event_subscription"]["previous"]
+    except Exception:
```

Comment:
Okay, but is there a better or more precise way to check for a specific RPC method?
`lightning-cli plugin list` is not very good, as it does not know the version of the plugin and we don't have plugin names but just full path names. So maybe we could parse `lightning-cli help` and look for the method name. If found use it. Then we can get rid of the `try/except Exception` and let it fail if the other plugin has a problem.

What do you think? If you change it please also do it for the call to the `feeadjust_would_be_nice` method ;)